src/engine/game/gui/ingame/equip.h

```diff
@@ -65,6 +71,53 @@ class Equipment : public GameGUI {
 
     std::shared_ptr<graphics::Texture> getItemFrameTexture(int stackSize) const;
     std::shared_ptr<graphics::Texture> getEmptySlotIcon(Slot slot) const;
+
+    struct Binding {
```

Comment:
As per the code style, declarations must go in the following order: constants, enums, structs/classes, variables, functions (`Binding` needs to be put right below `private:`).